content/actions/learn-github-actions/expressions.md

```diff
@@ -129,6 +129,14 @@ Returns `true` if `search` contains `item`. If `search` is an array, this functi
 
 `contains(github.event.issue.labels.*.name, 'bug')` returns whether the issue related to the event has a label "bug".
 
+#### Example matching a string list
```

Comment:
Personally, I'd rather this be part of `Example using an array`.

The current "Example using an array" is really quite strange.

```diff
 #### Example using an array

+ `contains(fromJson('["push", "pull_request"]'), github.event_name)` returns whether the `github.event_name` is any of `push`, `pull_request`. This is preferred over something like `github.event_name == "push" || github.event_name == "pull_request"`).
- `contains(github.event.issue.labels.*.name, 'bug')` returns whether the issue related to the event has a label "bug".
+ `contains(github.event.issue.labels.*.name, 'bug')` returns whether the issue related to the event has a label "bug". This works because `github.event.issue.labels.*.name` is converted into an array. See _insert reference to the magic that does this_.
```

Reply:
You have a point there. the current example there does not really use an intuitive "array" and I spend quite some time trying to get this work - but how do you call this actually? Is is not really "wildcard expression", but it seems to be  something that is somehow resolvable internally? I don't know what `contains()` really does internally. Would be nice if somebody with a bit more insight could comment here.

Reply:
It took me a dozen searches to find it.

Here's the winning cs search:
https://cs.github.com/github/docs/blob/7ada1b26cf14e242d78ddf021ab743158b6a87f2/content/actions/learn-github-actions/expressions.md?q=repo%3Agithub%2Fdocs+%22%60*%60%22+action+NOT+path%3A%2F%5Etranslations%5C%2F%2F+NOT+path%3A%2F%5Econtent%5C%2Fcodespaces%5C%2F%2F+NOT+path%3A%2F%5Edata%5C%2Frelease-notes%5C%2Fenterprise-server%5C%2F3-4%5C%2F%2F+NOT+path%3A%2F%5Edata%5C%2Frelease-notes%5C%2F%2F+NOT+path%3A%2F%5Elib%5C%2Fredirects%5C%2Fstatic%5C%2F%2F+NOT+path%3A%2F%5Econtent%5C%2Frest%5C%2Foverview%5C%2F%2F#L345

And here's the content:
https://docs.github.com/en/actions/learn-github-actions/expressions#object-filters

Reply:
So, that would _probably_ be:

```
See [Object filters](#object-filters).
```

Since, amusingly, we were already in the right file _sigh_.

Reply:
I've rebased and applied your suggestion. The "array" example is now "object filter" and my new stuff is "string array"

Reply:
And there seems a trailing whitespace my editor wants to remove. I've added a separate commit for this.

Reply:
question to native speakers: should it say "string array" or better change to "array of strings"?

Reply:
> So, that would _probably_ be:
> ```
> See [Object filters](#object-filters).
> ```

Added you suggestions

Reply:
`array of strings` (I checked this with someone else...)